lib/unwrappr/cli.rb

```diff
@@ -17,6 +17,13 @@ class CLI < Clamp::Command
            DESCRIPTION
            attribute_name: :base_branch)
 
+    option ['-f', '--lock-file'],
```

Comment:
Would this make more sense as a plural?

```suggestion
    option ['-f', '--lock-files'],
```

Reply:
Given the way to specify multiple gemfile args is to repeat the argument, I chose to keep it singular to indicate that users should provide one file per `--lock-file` arg.

Your suggestion on the usage docs should make that clear to users.